.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -42,18 +42,26 @@ If you need to submit a pr for any of these repos during this period, you should
 </details>
 
 - [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
-- [ ] **Disable nightly testing** to free up resources and ensure no competing jobs during release week
-- [ ] **Update aqaReference** to update with new git tag for 'aqaReference' to be used in release pipeline
-- [ ] **Run a trial release pipeline** to ensure less surprises on release day (typically against a milestone build)
+- [ ] **Disable nightly build completely** by disabling the pipelines temporarily to free up resources and ensure no competing jobs during release week
+- [ ] Tag the build repositories (temurin-build, ci-jenkins-pipelines, jenkins-helper) for the release level tooling: github/\<repo\>/releases, "Draft new release". Choose tag: "v\<YY\>.\<MM\>.01" (increment 01 if needed..)
```

Comment:
maybe use `v\<YYYY\>.\<MM\>.01` as the full year name

Reply:
+1